trace_context/HTTP_HEADER_FORMAT.md

```diff
@@ -145,16 +145,43 @@ base16(<TraceOptions>) = 00  // not-sampled
 
 # TraceState field
 
+The `tracestate` HTTP header field conveys information about request position in multiple distributed tracing graphs.
+
 ## Header name
 
 `tracestate`
 
 ## Header value
 
-`vendorName1=opaqueValue1,vendorName2=opaqueValue2`
+This section uses the Augmented Backus-Naur Form (ABNF) notation of [RFC5234](https://tools.ietf.org/html/rfc5234), including the DIGIT rule from that document. It also includes the OWS rule from [RFC7230](https://www.rfc-editor.org/info/rfc7230).
```

Comment:
Might be good to add some quick in-line layman definitions for `DIGIT` and `OWS` - they're not particularly complicated and would save some people not steeped in specs (like me) from having to go digging around in some lengthy RFCs. 

At the very least more focused links that get the reader closer to the thing you're pointing out, like [the DIGIT rule in appendix B.1 for RFC 5234](https://tools.ietf.org/html/rfc5234#appendix-B.1) and [the OWS rule from RFC 7230 Section 3.2.3](https://tools.ietf.org/html/rfc7230#section-3.2.3) (the RFC7230 link you currently have doesn't take you directly to the spec, let alone a section in the spec - I recommend this tools.ietf.org/html link instead even if you don't deep link into the relevant section).